conbench/templates/benchmark-result.html

```diff
@@ -184,12 +184,6 @@ <h3>History plot for benchmark '{{ benchmark.display_bmname }}'</h3>
               <div align="right" style="display:inline-block; float: right;">{{ run.commit.branch }}</div>
             </li>
           {% endif %}
-          {% if run.commit.fork_point_sha %}
-            <li class="list-group-item" style="overflow-y: auto;">
-              <b>forked from default at</b>
-              <div align="right" style="display:inline-block; float: right;">{{ run.commit.fork_point_sha }}</div>
-            </li>
-          {% endif %}
```

Comment:
This helps with #682 by simply removing these rows. We could come back to displaying fork point info if we're interested later

Reply:
I suspect we will want to reintroduce these sooner rather than later — it can be very helpful to understand why the baseline is what it is in this view, but it shouldn't stop merging this PR now if it's too much to keep i tin